README.md

```diff
@@ -1,8 +1,93 @@
----
-tags: 
-languages: 
-resources: 
----
-# Ttt 5 Move Rb
-
-*Idea: Give them #display_board so they can see output*
\ No newline at end of file
+# Tic Tac Toe CLI: Adding Player Move to the Game Board
+
+## Objectives
```

Comment:
These feel more like instructions. The Objectives are really:

define a method that updates an array passed to it.
use a method in a CLI
accept user input via gets
use user input in a method.